### DISCUSSION TITLE
[Feature] Add job that reconciles the library

### DISCUSSION THREAD

**Discussion by briandipalma** [immich-app/immich#3455] [Feature Request]
### The feature

If you add or delete a photo in a users library folder Immich will not notice it or will not remove it from the web view. There should be a way of triggering a reconciliation. This would allow Immich to upload photos from phones, import existing collections and manage these photos while letting people use programs like darktable to quickly sort and delete unwanted photos. Of course the photos would have to be deleted on the phones first else they'd just be synced back in again.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Closing in favour of the more active #4747
